frontend/src/concepts/pipelines/content/createRun/submitUtils.ts

```diff
@@ -69,6 +68,24 @@ const convertDateDataToKFDateTime = (dateData?: RunDateTime): DateTimeKF | null
   return date.toISOString();
 };
 
+export const convertPeriodicTimeToSeconds = (timeString: string): number => {
```

Comment:
Is this okay to have a utility function here? Or should I move this to another utility file?

Reply:
Two general notes:
- There is a `time.ts` file in `frontend/src/utilities` -- would it not serve better being there?
- When creating a local utility function to break up code from another function, you have no need to export it out of the file/module -- but I think this belongs in `time.ts` so just an educational point